de.fin_ger.OvGUCanteen.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id": "de.fin_ger.OvGUCanteen",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "gnome-ovgu-canteen",
+    "desktop-file-name-suffix": " (Flatpak)",
```

Comment:
```suggestion
```

Reply:
Oops, this was a testing suffix. Sorry, will remove it.